library/Cube/IcingaDb/CustomVariableDimension.php

```diff
@@ -20,9 +34,34 @@ class CustomVariableDimension implements Dimension
 
     public function __construct($name)
     {
+        if (count($result = explode('.vars.', $name)) === 2) {
```

Comment:
This may also lead to false positives if the customvar itself contains `.vars.`. Consider splitting this by a regex.